.gitignore

```diff
@@ -1,6 +1,7 @@
 /vendor
```

Comment:
Note: FileManager stopped working, and it could be a tricky one too, so we need a solution for it ASAP:

<img width="1510" alt="CleanShot 2023-01-03 at 18 00 03@2x" src="https://user-images.githubusercontent.com/1032474/210394195-46e3ad2e-e7f3-44f2-9280-622b5cc8dfaf.png">

Reply:
![youre-wrong-schitts-creek-gif-by-cbc](https://user-images.githubusercontent.com/1032474/210431005-9be95024-f683-42a3-8275-26f007aea030.gif)

FileManager does work, as long as you don't delete its CSS & JS assets. So... as long as you install it the same way you have until now. Which turns out... is the only way to use it... because I've tried to modernize it, and load everything through Bassets, and... we can't. Because the default elFinder theme... its css... includes links to images 🤦‍♂️

So... yeah. We can either leave it like it is... or find a way to modernize it too. See https://github.com/Laravel-Backpack/FileManager/pull/25 for more details.